wmn-data.json

```diff
@@ -6602,6 +6602,16 @@
         "m_code" : 404,
         "known" : ["carlito", "taylor"],
         "cat" : "social"
+      },
+      {
+        "name" : "Statuspage",
+        "uri_check" : "https://{account}.statuspage.io/api/v2/status.json",
+        "e_code" : 200,
+        "e_string" : "id",

```

Comment:
This `e_string` is a bit too plain for us as it could appear on page when a profile is not present. Consider replacing with `updated_at`